### ISSUE TITLE
Write Molecula tests

### ISSUE THREAD

**Issue by ArseniiPetrovich** [poanetwork/blockscout-terraform#137]
It might be a very good idea to write a number of Molecula tests to cover current deployment. Also we might consider writing the linting tests.

**Comment by vbaranov** [comment]
@ArseniiPetrovich could you post a link to what does Molecula represent?

**Comment by ArseniiPetrovich** [comment]
Yes, @vbaranov, sure. Here is the docs: https://molecule.readthedocs.io/en/stable/

Here is an example of successful integration this to POA bridges: https://github.com/poanetwork/tokenbridge

**Comment by ArseniiPetrovich** [comment]
Sorry, I misspelled the name a bit...
